openedx_ledger/admin.py

```diff
@@ -303,17 +330,24 @@ def get_readonly_fields(self, request, obj=None):
         """
         if obj and not obj._state.adding:  # pylint: disable=protected-access
             return ['ledger'] + self._readonly_fields
-        return self._readonly_fields
+        return []
 
     def get_fields(self, request, obj=None):
         """
         Don't include the ``quantity_usd_input`` field unless we're creating a new adjustment.
         """
-        # When we're adding a new adjustment, use default fields
         if not obj:
-            return super().get_fields(request, obj)
+            # When we're adding a new adjustment, show only the fields relevant for input.
+            return [
+                'ledger',
+                'quantity_usd_input',
+                'reason',
+                'notes',
+                'transaction_of_interest',
+            ]
```

Comment:
Create form for both Adjustments and Deposits should only show input fields relevant for creating a new Adjustment/Deposit, to avoid confusion.  Otherwise the default get_fields() just spews out a bunch of unrelated fields that I can't figure out how to hide programmatically.